app/components/chat/MessageForm.js

```diff
@@ -6,22 +6,42 @@ class MessageForm extends Component {
     super(props);
 
     this.onSendMessage = this.onSendMessage.bind(this);
+    this.onMessageSuccess = this.onMessageSuccess.bind(this);
+    this.onMessageError = this.onMessageError.bind(this);
   }
 
   componentDidMount(){
     ReactDOM.findDOMNode(this.refs.messageBox).focus();
   }
 
+  disableMessageBox(){
+    let box = $(ReactDOM.findDOMNode(this.refs.messageBox));
```

Comment:
That makes the input element on the message form a jquery-wrapped set. It just gives us the syntactic sugar for `.prop` and `.removeAttr`.